src/popup/components/AccountCard.vue

```diff
@@ -8,67 +8,67 @@
   >
```

Comment:
I think we should have 2 separate components for the regular account and the multisig account. But the deadline is near so I'm just leaving the comment for the future changes...